test/core/end2end/fixtures/h2_tls_common.h

```diff
@@ -51,7 +51,8 @@ struct SecurityPrimitives {
   enum VerifierType {
     EXTERNAL_SYNC_VERIFIER = 0,
     EXTERNAL_ASYNC_VERIFIER = 1,
-    HOSTNAME_VERIFIER = 2
+    HOSTNAME_VERIFIER = 2,
+    TEST_VERIFIER = 3,
```

Comment:
I'd prefer this verifier name being more descriptive of what it does, e.g. ROOT_SUBJECT_VERIFIER or so. It's test code so it can always be renamed if needed in the future.

Reply:
Not sure if you saw my comment in the other chain, copying here:
> I named the custom verifier TestExternalVerifier instead of something specific to root cert subject. I did this so that we have a modifiable custom verifier for checking any property if it is something we want to expand in the future.

Really, there's nothing that tests all of these properties yet, so this test verifier should get expanded to test all the different properties that we populate from `tsi_peer`

Reply:
I like that, will update